setup.py

```diff
@@ -6,4 +6,7 @@
     description="Miscellaneous utilities useful for interaction with CDIS systems.",
     license="Apache",
     packages=["cdisutils"],
+    install_requires=[
```

Comment:
hmm, was this not installing from `requirements.txt` properly? if it was, it might bet out of sync here because devs might not realize it's in both places